packages/bpk-component-chip-group/src/Nudger.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { type MutableRefObject, useEffect, useState } from 'react';
+
+import { BpkButtonV2, BUTTON_TYPES } from '../../bpk-component-button';
+import { CHIP_TYPES } from '../../bpk-component-chip';
+import { withButtonAlignment } from '../../bpk-component-icon/index';
+import ArrowLeft from '../../bpk-component-icon/sm/long-arrow-left';
+import ArrowRight from '../../bpk-component-icon/sm/long-arrow-right';
+import { cssModules, isRTL } from '../../bpk-react-utils/index';
+
+import type { ChipStyleType } from './BpkChipGroup';
+
+import STYLES from './Nudger.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+
+const CHIP_STYLE_TO_BUTTON_STYLE = {
+  [CHIP_TYPES.default]: BUTTON_TYPES.secondary,
+  [CHIP_TYPES.onDark]: BUTTON_TYPES.secondaryOnDark,
+  [CHIP_TYPES.onImage]: BUTTON_TYPES.primaryOnDark,
+}
+
+
+type Props = {
+  ariaLabel: string;
+  chipStyle?: ChipStyleType;
+  scrollContainerRef: MutableRefObject<HTMLElement | null>;
+  leading?: boolean;
```

Comment:
I would say rather than having this as a bool. I would use an enum that has leading or trailing in it.

Something like 
```  
  
const POSITION = {
   leading: 'leading',
   trailing: 'trailing'
}

position: Position;
```